live-examples/js-examples/typedarray/typedarray-tolocalestring.js

```diff
@@ -0,0 +1,10 @@
+const uint8 = new Uint32Array([500, 8123, 12]);
+
+console.log(uint8.toLocaleString());
+// expected output: "500,8123,12"
+
+console.log(uint8.toLocaleString('en-GB'));
+// expected output: "500,8,123,12"
+
+console.log(uint8.toLocaleString('de-DE', { style: 'currency', currency: 'EUR' }));
```

Comment:
```suggestion
console.log(uint8.toLocaleString("en-GB"));
// expected output: "500,8,123,12"

console.log(uint8.toLocaleString("de-DE", { style: "currency", currency: "EUR" }));
```

Reply:
Currently, all examples use single quotes in code, while the double quote is used in `expected output`. I think it's better to stick with that way for now and let this example be formatted later when we add prettier.

Reply:
Ah okay. Thanks for the info